### PR TITLE
Add native events types to simple editor widgets

### PR DESCRIPTION
**Pull request by pavelgruba**

### CODE REVIEW

File: js/ui/popup.d.ts [hunk 1]
```diff
@@ -55,13 +54,13 @@ export type InitializedEvent = InitializedEventInfo<dxPopup>;
 export type ShownEvent = EventInfo<dxPopup>;
 
 /** @public */
-export type ResizeEvent = NativeEventInfo<dxPopup> & ResizeInfo;
+export type ResizeEvent = EventInfo<dxPopup> & ResizeInfo;
```
Comment (DokaRus) [inline js/ui/popup.d.ts]:
```suggestion
export type ResizeEvent = NativeEventInfo<dxPopup, MouseEvent | TouchEvent> & ResizeInfo;
```

`EventInfo` and `ResizeInfo` does not contains the `event` field

File: js/ui/radio_group.d.ts [hunk 2]
```diff
@@ -29,7 +29,7 @@ export type InitializedEvent = InitializedEventInfo<dxRadioGroup>;
 export type OptionChangedEvent = EventInfo<dxRadioGroup> & ChangedOptionInfo;
 
 /** @public */
-export type ValueChangedEvent = NativeEventInfo<dxRadioGroup> & ValueChangedInfo;
+export type ValueChangedEvent = NativeEventInfo<dxRadioGroup, MouseEvent | PointerEvent> & ValueChangedInfo;
```
Comment (DokaRus) [inline js/ui/radio_group.d.ts]:
```suggestion
export type ValueChangedEvent = NativeEventInfo<dxRadioGroup, MouseEvent | PointerEvent | KeyboardEvent> & ValueChangedInfo;
```

**Review by DokaRus — DISMISSED** [review]
lgtm

File: js/ui/editor/editor.d.ts [hunk 3]
```diff
@@ -32,7 +32,7 @@ export interface EditorOptions<TComponent> extends WidgetOptions<TComponent> {
      * @action
      * @public
      */
-    onValueChanged?: ((e: NativeEventInfo<TComponent> & ValueChangedInfo) => void);
+    onValueChanged?: ((e: NativeEventInfo<TComponent, KeyboardEvent | MouseEvent | PointerEvent | TouchEvent | UIEvent> & ValueChangedInfo) => void);
```
Comment (DokaRus) [inline js/ui/editor/editor.d.ts]:
```suggestion
    onValueChanged?: ((e: NativeEventInfo<TComponent, KeyboardEvent | MouseEvent | PointerEvent | TouchEvent | Event> & ValueChangedInfo) => void);
```
For example, [change](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/change_event): `ChangeEvent: Event`

**Review by DokaRus — DISMISSED** [review]

**Review by DokaRus — APPROVED** [review]

**Review by IlyaKhD — APPROVED** [review]

**Pull request merged**